builtin/mainmenu/tab_online.lua

```diff
@@ -311,7 +333,15 @@ local function main_button_handler(tabview, fields, name, tabdata)
 
 		serverlistmgr.delete_favorite(server)
 		-- the server at [idx+1] will be at idx once list is refreshed
-		set_selected_server(tabdata, idx, tabdata.lookup[idx+1])
+		set_selected_server(tabdata, idx, tabdata.lookup[idx + 1])
```

Comment:
1. unrelated formatting change (but might be acceptable since it's just one)
2. why isn't the comment the line above updated too, now it's inconsistent :(

Reply:
Formatted it according to the style of the comment.